### PR TITLE
Refactor tiles and radio inputs to allow passing props down

### PR DESCRIPTION
**Pull request by salomao-rodrigues**

### CODE REVIEW

File: src/elements/radio-input/src/index.tsx [hunk 1]
```diff
@@ -5,66 +5,27 @@ import { jsx } from '@emotion/core'
...
-  )
-
 export type Width = 'half' | 'full'
 
 export const RadioInput: React.FC<Props> = ({
-  dataProps = {},
   label,
-  name,
-  onBlur,
   onChange,
```
Comment (peterhorne) [inline src/elements/radio-input/src/index.tsx]: Shouldn't `label` and `onChange` be passed down?

File: src/elements/tile-input/src/index.tsx [hunk 2]
```diff
@@ -5,43 +5,32 @@ import { jsx } from '@emotion/core'
...
-          <span css={st.childrenWrapper}>{children}</span>
-          {label}
-        </span>
+  ...inputProps
+}) => (
+  <label css={st.label}>
+    <input
+      css={st.input}
+      type="radio"
+      checked={selected === inputProps.value}
```
Comment (peterhorne) [inline src/elements/tile-input/src/index.tsx]: We've removed this behaviour from `RadioInput` should we also remove it from here?

Comment (salomao-rodrigues) [reply src/elements/tile-input/src/index.tsx]: Yes good catch

File: src/elements/radio-input/src/index.tsx (hunk 1 above)
Comment (salomao-rodrigues) [reply src/elements/radio-input/src/index.tsx]: `onChange` yes! gets a bit tricky to test onChange in storybook.
`label` makes sense to extract as it's only used in the span and not a property of input.

Comment (salomao-rodrigues) [reply src/elements/radio-input/src/index.tsx]: > gets a bit tricky to test onChange in storybook
Apparently I'm wrong, I'll add this

File: src/elements/radio-input/src/index.tsx [hunk 3]
```diff
@@ -5,66 +5,26 @@ import { jsx } from '@emotion/core'
...
 
 export const RadioInput: React.FC<Props> = ({
-  dataProps = {},
   label,
-  name,
-  onBlur,
-  onChange,
-  onFocus = () => {},
-  selected,
   value,
```
Comment (peterhorne) [inline src/elements/radio-input/src/index.tsx]: `value` needs to be passed down too 😄

Comment (salomao-rodrigues) [reply src/elements/radio-input/src/index.tsx]: 🤦‍♂

Comment (salomao-rodrigues) [reply src/elements/radio-input/src/index.tsx]: That might help with things working

**Pull request merged**